recipe/bld.bat

```diff
@@ -0,0 +1,37 @@
+@echo off
+
+mkdir %SRC_DIR%\_bootstrap
+msiexec /a %MSI_FILE% /qb TARGETDIR="%SRC_DIR%\_bootstrap"
+if %ERRORLEVEL% neq 0 exit /b %ERRORLEVEL%
+
+set "INSTALL_ROOT=%SRC_DIR%\_bootstrap\PFiles\Steel Bank Common Lisp"
+copy "%INSTALL_ROOT%\sbcl.exe" "%INSTALL_ROOT%\sbcl" > nul
+if %ERRORLEVEL% neq 0 exit /b %ERRORLEVEL%
+
+set "SBCL_HOME=%INSTALL_ROOT%"
+set "PATH=%SBCL_HOME%;%PATH%"
+
+cd %SRC_DIR%\sbcl-source
+  set "PATH=%BUILD_PREFIX%\Library\mingw-w64\bin;%PATH%"
+  set "CC=gcc"
```

Comment:
If you really use `gcc` as the compiler, you should be using a different `stdlib` and `compiler` function (i.e. `m2w64_c`) in the meta.yml for windows.

Reply:
Yes, `gcc` is actually hard-coded in one of the config files. I used  that compiler initially, but it failed some compilation. Let me give it another try

Reply:
It seems to me that when I don't specify `m2w64_c` the `gcc` that is used is `12.x` (https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=933100&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=6119ccbe-9301-594f-7c27-f792b80a7fcc&l=991)
How to specify this more recent gcc version?

Reply:
I actually don't know where this gcc is coming from.

Reply:
I tried to make the recipe depend on the `m2w64-experimental` using `channel_sources` and `channel-targets`, but it does not seem to work. It feels like we are at a dead-end?